src/server/src/main/java/io/cassandrareaper/service/SegmentRunner.java

```diff
@@ -124,6 +127,9 @@
     this.repairUnit = repairUnit;
     this.repairRunner = repairRunner;
     this.segmentFailed = new AtomicBoolean(false);
+    this.gotCompleteNotification = new AtomicBoolean(false);
+    this.gotFailedNotification = new AtomicBoolean(false);
+    this.gotSuccessNotification = new AtomicBoolean(false);
```

Comment:
can we remove these 3 lines please, they are not needed (the fields can be initialised on lines 99-101)

Reply:
ok, but I was just being consistent with the way we deal with `segmentFailed` ;)